src/sass_context.cpp

```diff
@@ -375,6 +375,11 @@ extern "C" {
     return options;
   }
 
+  const char* ADDCALL sass_copy_c_string (const char* input_string)
```

Comment:
Not 100% sure, but I feel `char*` without const would be more appropriate here, since you are indeed responsible to delete (and therefore change) the memory. Thinking about it, we may should also provide a `sass_free_c_string` function then?